src/common/types/store-data/unified-data-interface.ts

```diff
@@ -52,11 +52,21 @@ export interface UnifiedResult {
     uid: string;
     status: InstanceResultStatus;
     ruleId: string;
-    identifiers: StoredInstancePropertyBag;
+    identifiers: UnifiedIdentifiers;
     descriptors: StoredInstancePropertyBag;
-    resolution: StoredInstancePropertyBag;
+    resolution: UnifiedResolution;
 }
 
+export type UnifiedIdentifiers = {
+    id: string;
+    shortId: string;
+    identifierName: string;
+} & InstancePropertyBag;
+
+export type UnifiedResolution = {
+    howToFixSummary: string;
+} & InstancePropertyBag;
+
```

Comment:
we should be careful which fields we add to ensure this doesn't become a union of 3 different platform property bags